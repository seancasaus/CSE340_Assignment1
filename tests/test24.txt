kFAjf (j)|((1).(x)) , Jz (((k)|(8))|(f)).((e)*) , b9MGS (((O)|(B))*).(((s)|(c)).((G).(o))) , Zv ((8)*).((6).((q).(z))) , X72YJ (Q)|((6)|(w)) #
"886qz886qzBcGo686qz fjfee6qz86qzf86qz6Q OcGocGo8e86qzj " 
