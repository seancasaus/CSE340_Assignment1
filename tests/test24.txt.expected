Zv , "886qz"
Zv , "886qz"
b9MGS , "BcGo"
X72YJ , "6"
Zv , "86qz"
Jz , "f"
kFAjf , "j"
Jz , "fee"
Zv , "6qz"
Zv , "86qz"
Jz , "f"
Zv , "86qz"
X72YJ , "6"
X72YJ , "Q"
b9MGS , "OcGo"
b9MGS , "cGo"
Jz , "8e"
Zv , "86qz"
kFAjf , "j"
